xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/results/HuobiDepthResult.java

```diff
@@ -0,0 +1,35 @@
+package org.knowm.xchange.huobi.dto.marketdata.results;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Date;
+import org.knowm.xchange.huobi.dto.HuobiResult;
+import org.knowm.xchange.huobi.dto.marketdata.HuobiDepth;
+
+public class HuobiDepthResult extends HuobiResult<HuobiDepth> {
+
+  private final Date ts;
+  private final String ch;
+
+  @JsonCreator
+  public HuobiDepthResult(
+      @JsonProperty("status") String status,
+      @JsonProperty("ts") Date ts,
+      @JsonProperty("tick") HuobiDepth tick,
+      @JsonProperty("ch") String ch,
+      @JsonProperty("err-code") String errCode,
+      @JsonProperty("err-msg") String errMsg) {
+    super(status, errCode, errMsg, tick);
+    this.ts = ts;
+    this.ch = ch;
+    getResult().setTs(ts);
```

Comment:
this is odd, these DTO object should represent the exchanges response and should not do any premature mapping nor have data coping logic

Reply:
Also copied. I am not very sure about the usage. Removed them in next change.